benchmark/matmul_benchmark.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+import time
+
+import torch
+import torch.nn.functional as F
+from aqlm.cuda.cuda_kernel import CUDA_KERNEL
+from aqlm.utils import _dequantize_weight, pack_int_data, unpack_int_data
+from torch import nn
+
+
+def benchmark(f, warmup=10, iter=10):
+    for i in range(warmup + iter):
+        f()
+        if i == warmup - 1:
+            torch.cuda.synchronize()
+            tick = time.perf_counter()
+    torch.cuda.synchronize()
+    average_latency = (time.perf_counter() - tick) / iter
+    time.sleep(1.0)
+    return average_latency
+
+
+MODELS = {
+    "fuse": {
+        "Llama 2 7B": [(4096, 3 * 4096), (4096, 4096), (4096, 2 * 11008), (11008, 4096)],
+        "Llama 2 13B": [(5120, 3 * 5120), (5120, 5120), (5120, 2 * 13824), (13824, 5120)],
+        "Llama 2 70B": [(8192, int(1.25 * 8192)), (8192, 8192), (8192, 2 * 28672), (28672, 8192)],
+    },
```

Comment:
a bit confusing to first time reader, consider 
- adding a short comment line explaining how this works 
- removing what is certainly not used

OR add to todo for next PR

Reply:
I've deleted the unnecessary code and added some comments